dashboards/snyk/snyk.json

```diff
@@ -0,0 +1,289 @@
+{
```

Comment:
I'm seeing some accountIds in here, let's get this json cleaned up. The `sanitize-dashboard` script should handle that for you, instructions [here](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards)

Reply:
@aswanson-nr sorry for the delay, dashboard should be sanitized now